.github/workflows/ci.yaml

```diff
@@ -57,6 +57,6 @@ jobs:
       - name: Run Tests
         if: matrix.subgraph == 'polygon-digital-carbon' || matrix.subgraph == 'carbonmark' || matrix.subgraph == 'pairs'
         env:
-          GITHUB_TOKEN: ${{ secrets.GH_ACCESS_TOKEN }} 
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }} 
```

Comment:
Why this change?

Reply:
i had the incorrect key from the docs; correct one is GH_TOKEN

these are still occasionally timing out; i'm assuming we can improve the workflow by caching the dependencies but should be in a follow up PR